scripts/windows/build/rocksdb-win-arm32.ps1

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+function Get-Rocksdb-WinArm32
+{
+    $ErrorActionPreference = 'Continue'
+
+    $vcpkgroot = "$env:HOMEDRIVE\vcpkg1"
```

Comment:
Extract the vcpkg-installation code from `Get-OpenSSL` into a function and use that here instead of this. (Of course you will need that function to take a parameter whether to checkout master or specific commit.)